README.md

```diff
@@ -1 +1,5 @@
-# model-service
```

Comment:
would keep this header, since we should still later add some introductory stuff in the readme later anyway

Reply:
done